bin/format_exercises

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Format existing exercises using rustfmt
+
+EXERCISES_PATH="../exercises/"
```

Comment:
This will only work if you are in the `bin` directory when you run this script. Please change it to base the path off the repo root. See [here](https://github.com/exercism/rust/blob/master/_test/check-configlet-fmt.sh#L6) for an example of how to do that.